src/query/functions/src/scalars/geometry.rs

```diff
@@ -67,6 +74,81 @@ pub fn register(registry: &mut FunctionRegistry) {
         })
     );
 
+    registry.register_passthrough_nullable_2_arg::<GeometryType, GeometryType, GeometryType, _, _>(
+        "st_makeline",
+        |_, _, _| FunctionDomain::Full,
+        vectorize_with_builder_2_arg::<GeometryType, GeometryType, GeometryType>(
+            |left_exp, right_exp, builder, ctx| {
+                if let Some(validity) = &ctx.validity {
+                    if !validity.get_bit(builder.len()) {
+                        builder.commit_row();
+                        return;
+                    }
+                }
+
+                let geometries: Vec<Geometry> = match binary_to_geometry(&vec![left_exp, right_exp])
```

Comment:
Seems we need impl an `Into<Geometry>` to simply convert.

Reply:
I I simulate the implementation of PostGIS [code] (https://github.com/postgis/postgis/blob/88c3893c718847867714dc1bfddfa5db7c0a55e3/liblwgeom/lwline.c#L182C1-L203C4); The [functions](https://postgis.net/docs/ST_MakeLine.html) define in postgis are more flexible than snowflake;
To Implement Into<Geometry> function , we must include geozero crate in georust/geo project to process wkb logic, I'm not sure it's accepted by the community.

Reply:
Seems impl [this trait](https://docs.rs/geozero/latest/geozero/trait.ToGeos.html) is enough.

Reply:
Although postgis maybe more flexible, we should follow the Snowflake way, Databend is a Snowflake alternative.

Reply:
We need commit in geozero for trait ToGeos further.